packages/components/pie-toast/src/defs.ts

```diff
@@ -2,6 +2,8 @@ import { type ComponentDefaultProps } from '@justeattakeaway/pie-webc-core';
 
 export const variants = ['neutral', 'info', 'warning', 'success', 'error'] as const;
 
+export const defaultDuration = 5000;
```

Comment:
I think this should be moved to `defaultProps` instead

Reply:
Agreed!